src/amo/components/AddonSummaryCard/index.js

```diff
@@ -56,7 +70,17 @@ export const AddonSummaryCardBase = ({
       </div>
       <div className="AddonSummaryCard-header-text">
         <h1 className="visually-hidden">{headerText}</h1>
-        <AddonTitle addon={addon} linkToAddon linkSource={location.query.src} />
+        <AddonTitle
+          addon={addon}
+          linkToAddon
+          linkSource={
+            // `queryParamsForAttribution` is the return value of
+            // `getQueryParametersForAttribution()`, which can either return
+            // `utm_content` or `src`.
```

Comment:
why have two different parameter values?  Wouldn't a single generically named property be easier?

Reply:
That's because of the feature flag. It was just easier to do this than importing some `config` and checking the flag.